services/entry.js

```diff
@@ -0,0 +1,14 @@
+const { Entry } = require('../models')
+
+exports.getEntries = async () => {
+  try {
+    const entries = await Entry.findAll({
+      attributes: {
+        exclude: ['id', 'content', 'categoryId', 'type', 'updatedAt', 'deletedAt'],
```

Comment:
I think... we need id, content, categoryId, type included in results....

Reply:
Ready!